app/views/entries/index.html.erb

```diff
@@ -28,8 +28,13 @@
             <%= content_tag :td, entry.author, class: 'table__cell' %>
             <%= content_tag :td, entry.published_at.present? ? l(entry.published_at) : t('.published_at_unknown'), class: 'table__cell' %>
             <%= content_tag :td, entry.read?, class: 'table__cell' %>
-            <%= content_tag :td, class: 'table__cell' do %>
-              <%= link_to t('.show'), entry, class: 'table__link' %>
+            <%= content_tag :td, class: 'table__cell table__cell--actions' do %>
+              <%= link_to entry, class: 'table__link', 'aria-label': t('.show') do %>
+                <%= inline_svg_tag 'images/icons/eye.svg', title: t('.show'), class: 'icon' %>
```

Comment:
```suggestion
                <%= inline_svg_tag 'images/icons/eye.svg', class: 'icon' %>
```
We can leave this attribute out, since no one would be able to see the `<title>...</title>` inside the svg anyway

Reply:
It's visible as a browser tooltip (which is why I'd prefer to leave it in).

Reply:
In that case we really don't want that, since: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns
We either want: (1) a text visible to all users (in which case we can hide the icon with `aria-hidden`), or (2) a proper tooltip, or (3) just something for assistive technologies. 
I'd prefer 1 or 3 for simplicity